### PR TITLE
[TT-10135] update chart version for release

### PR DESCRIPTION
**Pull request by komalsukhani**
## Description

1. Updated chart version as follows:
- tyk-oss: 1.0.0 → 1.1.0
- tyk-data-plane: 1.0.0
- tyk-stack: 1.0.0-beta1
- tyk-gateway: 1.0.0 → 1.1.0
- tyk-pump: 1.0.0 → 1.1.0
- tyk-bootstrap: 1.0.0-beta5 → 1.0.0-beta6
- tyk-dashboard: 1.0.0-beta5 → 1.0.0-beta6
- tyk-dev-portal: 1.0.0-beta1
- tyk-enterprise-portal: 1.0.0-beta4 -> 1.0.0-beta5
- tyk-mdcb-data-plane: 1.0.0 -> 1.1.0
- tyk-single-dc: 1.0.0-beta6 -> 1.0.0-beta7

2. Fixed this [comment](https://github.com/TykTechnologies/tyk-charts/pull/144#discussion_r1375828414)
I misunderstood it. That need to be fixed.

## Related Issue

[TT-10135](https://tyktech.atlassian.net/browse/TT-10135)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] Documentation updates or improvements.

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [x] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
     - [ ] I have manually updated the README(s)/documentation accordingly.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

[TT-10135]: https://tyktech.atlassian.net/browse/TT-10135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-single-dc/values.yaml
Comment (buraksekili) [inline tyk-single-dc/values.yaml:1]: could you please provide some insight into the reason for reverting the changes made to this file?

Comment (buraksekili) [reply tyk-single-dc/values.yaml:1]: oh, i see https://github.com/TykTechnologies/tyk-charts/pull/144#discussion_r1375828414 from PR description

Comment (komalsukhani) [reply tyk-single-dc/values.yaml:1]: Yeah. For some reason I thought @andrei-tyk was suggesting those changes. 
But later I realised that he was asking to undo them.
Sorry I misunderstood it.

Comment (komalsukhani) [reply tyk-single-dc/values.yaml:1]: Ohh wait. Since we are using new bootstrap image in single-dc, we need to have these changes right?

Comment (komalsukhani) [reply tyk-single-dc/values.yaml:1]: @buraksekili I updated single-dc to use tyk-dev-portal chart. Now it is working fine

**Review by buraksekili — APPROVED** [review]

**Pull request merged**